src/QueryBuilder/ResultViewer/ResultViewer.css

```diff
@@ -0,0 +1,19 @@
+.ResultViewerAccordionHeader {
+  display: flex;
+  gap: 0.25em;
+  justify-content: space-between;
+  width: 100%;
+  /* 3.5 is a bit arbitrary, but covers the chevron and scrollbar in my testing. */
+  /* it's not really possible to get the true width due to how nested this element is :( */
+  max-width: calc(100vw - 3.5em);
```

Comment:
I don't love this, but I can't manage to get anything to work properly with `width: 100%`. The QB accordion is many levels of relative/flex elements deep, so any actual meaning of the view's width is lost.